### PR TITLE
Attempt to clarify this test

### PR DESCRIPTION
**Pull request by PeterJCLaw**

### CODE REVIEW

File: controllers/territory_controller/tests.py [hunk 1]
```diff
@@ -216,40 +231,34 @@ def setUp(self, _: object, __: object, ___: object) -> None:
...
+        self.claim_territory(StationCode.PN, Claimant.ZONE_1)
+        self.claim_territory(StationCode.EY, Claimant.ZONE_1)
+
+        self.assertNotLocked(StationCode.PN, "early after second claims")
+        self.assertNotLocked(StationCode.EY, "early after second claims")
+
+        self.claim_territory(StationCode.PN, Claimant.ZONE_0)
+
+        self.assertLocked(StationCode.PN, "after third claim")
+        self.assertNotLocked(StationCode.EY, "after third claim")
```
Comment (WillB97) [inline controllers/territory_controller/tests.py:261]: We haven't actually claimed this
```suggestion
        self.assertNotLocked(StationCode.EY, "after lock-out of PN")
```

File: controllers/territory_controller/tests.py [hunk 2]
```diff
@@ -216,40 +231,34 @@ def setUp(self, _: object, __: object, ___: object) -> None:
...
+    @patch('territory_controller.TERRITORY_LINKS', new={
+        (StationCode.PN, StationCode.EY),
+        (TerritoryRoot.z0, StationCode.PN),
+        (TerritoryRoot.z1, StationCode.PN),
+    })
     def test_territory_lockout(self, _: object) -> None:
         """
         Test a territory is locked after the correct number of claims and
         disconnected territories aren't also locked
         """
```
Comment (WillB97) [inline controllers/territory_controller/tests.py:244]:
```suggestion
        EY is only capturable when PN is owned and
        has its ownership reset when PN's owner changes
        """
```

**Review by WillB97 — APPROVED** [review]
While this hardcodes the `LOCKED_OUT_AFTER_CLAIM` value it makes things more obvious

File: controllers/territory_controller/tests.py (hunk 2 above)
Comment (PeterJCLaw) [reply controllers/territory_controller/tests.py:254]: Does 8d30602 convey this?

Comment (WillB97) [reply controllers/territory_controller/tests.py:254]: Yes, that works.

**Review by WillB97 — APPROVED** [review]

**Comment by PeterJCLaw** [comment]
> While this hardcodes the `LOCKED_OUT_AFTER_CLAIM` value it makes things more obvious

How do you mean? The test also mocks out the value, so the coupling is all within the test itself.

**Pull request merged**